subjects/error_types/README.md

```diff
@@ -5,6 +5,7 @@
 For this exercise, you will have to implement an **error type** for a form validator. This must validate the password and the first name.
 
 The first name must not be empty and the password must have **at least 8 characters**, and a combination of **alphabetic**, **numeric** and **none-alphanumeric** (`<`, `&`,  `/` ...).
+Additionally, you need to create a helper function create_date that converts a string like "2015-09-05" into a NaiveDate format.
```

Comment:
The question for this exercise to build the `error type` form validator, the data will be given as input in the right format.
they can use `NaiveDate::parse_from_str(date_str, "%Y-%m-%d")` if needed for test so no need to create a function to converts the date